css/style.css

```diff
@@ -0,0 +1,866 @@
+  margin: 0 auto;
+  margin-top: 50px;
+}
+
+.text-left {
+  margin-bottom: 50px;
+}
+
+.review-score {
+  width: 300px;
+}
+
+.review-title {
+  margin-top: 49px;
+  margin-bottom: 26px;
+}
+
+.score,
+.review-security {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 90px;
+}
+
+.score__list {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 21px;
+  text-transform: uppercase;
+}
+
+.score__list span {
+  font-weight: bold;
+  font-size: 10px;
+  line-height: 12px;
+}
+
+li {
+  list-style-type: none;
+  margin-bottom: 16px;
+}
+
+ul {
+  margin: 0;
+  padding-left: 0;
+}
+
+.stars {
+  margin-bottom: 12px;
+  display: flex;
+}
+
+.security {
+  background: url(../img/security.svg) center top no-repeat;
+  margin-right: 20px;
+  width: 62px;
+  height: 74px;
+}
+
+.review-security {
+  justify-content: center;
+}
+
+.security-title {
+  font-weight: 500;
+  font-size: 50px;
+  line-height: 55px;
+  margin: 0;
+}
+
+.security-subtitle {
+  font-weight: 500;
+  font-size: 25px;
+  line-height: 30px;
+  margin: 0;
+}
+
+.footer__logo {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  margin-right: 18px;
+  margin-bottom: 12px;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    margin: 24px 33px 85px 15px;
+  }
+
+  .inner {
+    justify-content: space-around;
+  }
+
+  .header__link,
+  .header__link_active {
+    display: none;
+  }
+
+  .menu {
+    justify-content: flex-end;
+  }
+
+  .menu__icon {
+    width: 35.23px;
+    height: 22.02px;
+  }
+
+  .burger {
+    width: 100%;
+  }
+
+  .intro {
+    height: 865px;
+  }
+
+  .intro__wrapper {
+    background-position: center center;
+  }
+
+  .intro__subtitle {
+    margin-bottom: 87px;
+  }
+
+  .intro__img {
+    margin: 0 47px 48px 47px;
+  }
+
+  .intro-info {
+    height: 190px;
+    width: 500px;
+    margin: 0 auto;
+    margin-bottom: 63px;
+  }
+
+  .intro-info__inner {
+    flex-wrap: wrap;
+    align-content: space-between;
+    justify-content: center;
+    height: 100%;
+  }
+
+  .feature {
+    width: 488px;
+    margin: 0 auto;
+  }
+
+  .info-title {
+    display: flex;
+    align-items: center;
+  }
+
+  .info-title > span {
+    font-size: 50px;
+    line-height: 61px;
+    margin-right: 9px;
+  }
+
+  .icon01 {
+    background-size: cover;
+    width: 66px;
+    height: 50px;
+    margin-right: 20px;
+  }
+
+  .icon02 {
+    background-image: url(../img/road.svg);
+    background-position: center center;
+    background-size: cover;
+    width: 44px;
+    height: 72px;
+    margin-right: 16px;
+  }
+
+  .icon03 {
+    background-image: url(../img/wheel.svg);
+    background-size: cover;
+    width: 30px;
+    height: 30px;
+    margin-right: 10px;
+  }
+
+  .order-title {
+    font-size: 16px;
+    line-height: 20px;
+    margin-right: 35px;
+    color: #E51937;
+    text-shadow: none;
+  }
+
+  .btn {
+    width: 220px;
+    height: 55px;
+    border: none;
+    color: #fff;
+    background: #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/finalproject/pullRequest?prid=5950407)